birdhouse/optional-components/prometheus-log-parser/config/thredds/prometheus-log-exporter.py

```diff
@@ -42,7 +42,7 @@
     r'\"(?P<user_agent>[^\"]+)\"\s'
     r'\"(?P<forward_for>[^\"]+)\"')
 
-LABEL_KEYS = ("remote_addr", "tds_service", "dataset")
+LABEL_KEYS = ("remote_addr", "tds_service", "dataset", "variable")
```

Comment:
"variable" isn't a group name in the regex. Did you mean to add it here?

Reply:
I mean to add it to the Counter label, but it'd be complicated to have it in the regex.